README.md

```diff
@@ -101,7 +101,7 @@ The following are the tags supported at this very moment:
 
 - For a parameter: `## @param fullKeyPath [modifier?] Description`.
 - For a section: `## @section Section Title"`.
-- To skip an object and all its children: `## @skip fullKeyPath`.
+- To skip an object and all its children: `## @skip fullKeyPath [Description?]`.
```

Comment:
Please note that in this comment (paying attention to the comments above) seems that the Description has to be between `[]`.

Also the regexp will allow lines like:
```
## @skip fullKeyPath [modifier?] Description
```
I think the modifier is not needed in this case but there is no problem to include it

Reply:
The modified doesn't make sense here. I just updated the regex and the documentation.